WalletWasabi.Gui/Converters/FeeTargetTimeConverter.cs

```diff
@@ -11,7 +11,7 @@ public class FeeTargetTimeConverter : IValueConverter
 	{
 		public static string Convert(int feeTarget, string minutesLabel, string hourLabel, string hoursLabel, string dayLabel, string daysLabel)
 		{
-			if (feeTarget is >= Constants.TwentyMinutesConfirmationTarget and <= 6) // minutes
+			if (feeTarget <= 6) // minutes
```

Comment:
Can we leave this as it was and add a new `if(feeTarget == 1){return "fastest"}`

Reply:
ok